assets/monitoring/README.md

```diff
@@ -6,13 +6,14 @@ Teleport exports Prometheus Metrics endpoints.
 
 To start teleport with prometheus endpoint enabled:
 
-```
-teleport start --diag-addr=127.0.0.1:3434
+```bash
+teleport start --diag-addr=127.0.0.1:3000
+# http://127.0.0.1:3000/metrics
 ```
 
 To start monitoring stack, simply `docker-compose up`
 
-Then go to `http://localhost:3000`
+Then go to `http://localhost:3001`
```

Comment:
Why a different port in `docker-compose` versus running directly on the machine?

Reply:
I did change it, https://github.com/gravitational/teleport/pull/4069/files#diff-f15524a82953c688d5d2f7086398011eR24 I did this to run Teleport Diagnostics on it's default port; when running locally, but let me know if there is a better way to do this.